src/__tests__/stopping.test.ts

```diff
@@ -0,0 +1,739 @@
+import { Cat } from '..';
+import { CatMap } from '../type';
+import {
+  EarlyStopping,
+  StopAfterNItems,
+  StopAfterNItemsInput,
+  StopIfSEMeasurementBelowThreshold,
+  StopIfSEMeasurementBelowThresholdInput,
+  StopOnSEMeasurementPlateau,
+  StopOnSEMeasurementPlateauInput,
```

Comment:
what is StopOnSEMeasurementPlateau?

Reply:
In simpler terms, if the SE doesn't significantly change for a set number of items, the process stops early because it indicates that the ability estimate is no longer improving. It's used to avoid unnecessary trials once the measurement has plateaued.

Reply:
See my comment here: https://github.com/yeatmanlab/jsCAT/pull/25#discussion_r1813477978